### PR TITLE
Re-link and review tasks 0-11

### PR DESCRIPTION
**Pull request by viktor-yakubiv**
1. Created plan of the course `contents.md`
2. Numbers of tasks are removed
3. Added meaningful links to tasks and link to _Contents_
4. Reviewed formatting of a few tasks
5. Added _Done_ section to last online task with greeting.

@OleksiyRudenko review, please. You can preview course UX at [viktor-yakubiv/kottans-frontend-2019](https://github.com/viktor-yakubiv/kottans-frontend-2019)

### CODE REVIEW

**Comment by viktor-yakubiv** [comment]
@OleksiyRudenko, probably we have merge conflict in 2 currently opened PRs. Order of merge is up to you.

I suggest quickly merge everything together and review it one more time afterwards.

**Comment by cidre** [comment]
typo: You finished Stage 0 of the cour**c**e

**Comment by OleksiyRudenko** [comment]
@viktor-yakubiv 
Have merged #103.
Please, see [comment on `contents.md` vs `tasks/README.md`](https://github.com/kottans/frontend/pull/103#issuecomment-429612242)

**Comment by viktor-yakubiv** [comment]
> typo: You finished Stage 0 of the cour**c**e

Thank you! Fixed

**Comment by OleksiyRudenko** [comment]
Would be good to have yet another reviewing pair of eyes.

File: contents.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+The online stage of _Front-End Course_. Following tasks should be completed
+at home to participate offline stage. Learn more in [README](README.md).
+
+### General Basics
+ 0. [Git Basics](tasks/git-intro.md)
+ 1. [Linux CLI and Networking](tasks/linux-cli-http.md)
+ 2. [VCS (hello gitty), GitHub and Collaboration](tasks/git-collaboration.md)
+
+### Front-End Basics
+ 3. [Intro to HTML&nbsp;&&nbsp;CSS](tasks/html-css-intro.md)
```
Comment (sudodoki) [inline contents.md]: does this need `&nbsp;` in here? Might be me just being grumpy, but strange to put non breaking space as entity and `&amp;` as `&` 🤔 No biggie, can merge as is.

**Comment by sudodoki** [comment]
Overall looks good. Not sure why contents.md instead of README.md in `tasks/` folder so it could be browsed through github directly. Also could just add README.md with link to `contents.md` and `README.md` as course description for easier navigation

**Comment by cidre** [comment]
Suggestion: to give more accurate and meaningful names for the stages. Instead of online/offline, аdmission: self-study stage and collaboration stage. Because  online/offline - are more about forms of work.

**Comment by suchov** [comment]
@viktor-yakubiv overall LGTM 💥  But check pls guy's suggestions when you'll have time and ping me to merge it then. Thanks!

**Comment by cidre** [comment]
typo: **The** following tasks should be completed at home to participate offline stage.
To be honest, this phrase is not accurate enough for me, because does not reflect the real situation with the tasks performed last year.

**Comment by OleksiyRudenko** [comment]
@sudodoki Currently `tasks/README.md` refers to all tasks in the folder including those that aren't a part of the course (not approved yet for inclusion or not placed in the syllabus).

`contents.md` is more about syllabus. Probably, it would make sense to rename it to `syllabus.md`.

File: contents.md (hunk 1 above)
Comment (viktor-yakubiv) [reply contents.md]: Thank you for suggestion. Just forgot about it

**Comment by viktor-yakubiv** [comment]
> `contents.md` is more about syllabus. Probably, it would make sense to rename it to `syllabus.md`.

I refer to _Contents_ in the all task. Should I rename links to _Go back to Syllabus_? What do you suggest on this point?

**Comment by viktor-yakubiv** [comment]
I rebased the branch and fixed issues

**Pull request merged**